test/endtoend/backup_restore_test.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+source ./tools/test.env
+source ./test/endtoend/utils.sh
+
+function get_started() {
+    echo "Apply latest operator.yaml"
```

Comment:
nit: I think `get_started()` can be shared between `backup_restore_test` and `upgrade_test`. The initial cluster manifest we want to apply could be passed down as an argument to `get_started()`

Reply:
Fixed!